pkg/catalog/outbound_traffic_policies.go

```diff
@@ -17,14 +17,14 @@ import (
 func (mc *MeshCatalog) ListOutboundTrafficPolicies(downstreamIdentity service.K8sServiceAccount) []*trafficpolicy.OutboundTrafficPolicy {
 	if mc.configurator.IsPermissiveTrafficPolicyMode() {
 		outboundPolicies := []*trafficpolicy.OutboundTrafficPolicy{}
-		mergedPolicies := trafficpolicy.MergeOutboundPolicies(outboundPolicies, mc.buildOutboundPermissiveModePolicies()...)
+		mergedPolicies := trafficpolicy.MergeOutboundPolicies(false, outboundPolicies, mc.buildOutboundPermissiveModePolicies()...)
```

Comment:
This is a very minor thing, which is always tough to fix, and nothing to delay a good PR over - but I always think if there is a better way than just dropping a `true` or `false` in a function.
One would have to go to the function definition to understand that this here means `allowPartialHostnamesMatch` is `false`.

One thing I'd try to do (in another PR, if you, @snehachhabria agree) - I'd create an enum and then pass something like `allowPartialHostnamesMatch` and `disallowPartialHostnamesMatch`.

Reply:
thanks for the suggestion I have made the changes in this PR itself